handlers/eth1tx.go

```diff
@@ -22,7 +22,7 @@ func Eth1TransactionTx(w http.ResponseWriter, r *http.Request) {
 
```

Comment:
The changes in this method don't relate to the task but while I had a look at the transaction view I noticed that some updates could be done here.

Reply:
This changes the behavior oh so slightly:
If an error is thrown, the first if caused a page title of "Transaction 123" while the second if caused a page title of "Transaction 0x123"
With your changes, both ifs write "Transaction 0x123".

To me, that's an improvement. But was that an intentional change by you?

And of course I am curious to know why you have extracted the `SetPageDataTitle` and the setting of `data.Meta.Path` out of the second if but not out of the first if.
Wouldn't it work to set both right above the first if?

Reply:
You are right, moved it up.
he important behaviour change is that *data.Meta.Path* now always contains the "0x" while before in the error cases it didn't.